### DISCUSSION TITLE
Get rid of empty space in pop-up card

### DISCUSSION THREAD

**Discussion by andreaconfa** [Clooos/Bubble-Card#949] [Questions about config, custom styles and templates]
Hi all,
how i can remove this empty space from a popup card?
![image](https://github.com/user-attachments/assets/31a2f3c1-a174-432a-b44a-7c074a22be9e)

thanks

**Comment by Clooos** [comment]
Hi! In the pop-up editor you can go to Styling options > Pop-up styling then there you can change the top margin options.

**Reply by andreaconfa** [reply]
> Hi! In the pop-up editor you can go to Styling options > Pop-up styling then there you can change the top margin options.

hi, changing the top margin property is moving my popup up or down, not reducing the popoup window width

**Reply by Clooos** [reply]
Ho I see, you want it to don't have any background below your cards, right? If it's what you want this is probably achievable with a custom style, but I'm also planning to add this as a feature in the future.

**Reply by andreaconfa** [reply]
> Ho I see, you want it to don't have any background below your cards, right? If it's what you want this is probably achievable with a custom style, but I'm also planning to add this as a feature in the future.

what i need is to end the card at the end of the last entity, keeping the popup in center of the screen.
how is possible with a Custom style?

**Reply by Clooos** [reply]
Yes that's what I thought, but this implies a complex custom style, so if you're not comfortable with CSS it might be difficult to achieve. But if you want to give it a try, take a look at the tips at the beginning of the Styling section in the documentation on GitHub, I explain where to start. 

Then don't hesitate to try to use ChatGPT or something similar, you can provide it the content of the pop-up styles.ts file, it can be powerful for that kind of things.

**Reply by andreaconfa** [reply]
> Yes that's what I thought, but this implies a complex custom style, so if you're not comfortable with CSS it might be difficult to achieve. But if you want to give it a try, take a look at the tips at the beginning of the Styling section in the documentation on GitHub, I explain where to start.
> 
> Then don't hesitate to try to use ChatGPT or something similar, you can provide it the content of the pop-up styles.ts file, it can be powerful for that kind of things.

tried with chatgpt and also editing varius property of the css but without success

**Reply by Clooos** [reply]
You can ask on the HA forum or on the Bubble Card subreddit, or you can wait that I add that as a new feature.

But I'm sorry, I'm not answering to complex custom styles requests, I don't have enough time for that, the Bubble Card development is already taking most of my daily spare time.

I hope you understand 🙂

**Comment by orestkos** [comment]
Maybe this option would work. Fixing along the bottom edge, and a margin is set at the top. I use this approach, it’s very convenient.

type: custom:bubble-card
card_type: pop-up
hash: "#shtora"
name: Curtain
icon: mdi:curtains
show_state: false
margin_top_mobile: calc(100vh - 540px)
margin_top_desktop: calc(100vh - 630px)
show_name: true
show_header: true

![1](https://github.com/user-attachments/assets/a0793fbb-33a1-46c2-aa26-b4a39355d6d2)
